canvas_modules/common-canvas/src/toolbar/toolbar-action-item.jsx

```diff
@@ -291,7 +292,8 @@ ToolbarActionItem.propTypes = {
 	toolbarActionHandler: PropTypes.func.isRequired,
 	instanceId: PropTypes.number.isRequired,
 	overflow: PropTypes.bool,
-	onFocus: PropTypes.func
+	onFocus: PropTypes.func,
+	buttonSize: PropTypes.oneOf(["md", "sm"])
```

Comment:
Call this prop just 'size' -- the same as it is everywhere else.

Reply:
Updated buttonSize with size everywhere.
Create test for both `sm` & `md`.
Adjusted comments & spaces

Reply:
Updated it.